kommunicate/src/main/java/io/kommunicate/KmConversationHelper.java

```diff
@@ -358,6 +359,18 @@ public void onFailure(RegistrationResponse registrationResponse, Exception excep
     public static void launchConversationIfLoggedIn(Context context, KmCallback callback) {
         if (Kommunicate.isLoggedIn(context)) {
             KmConversationBuilder conversationBuilder = new KmConversationBuilder(context);
+            if(KmDefaultSettingPreference.getInstance().getDefaultBotIds() != null) {
+                conversationBuilder.setBotIds(new ArrayList<String>(KmDefaultSettingPreference.getInstance().getDefaultBotIds()));
+            }
+            if(KmDefaultSettingPreference.getInstance().getDefaultAgentIds() != null) {
+                conversationBuilder.setAgentIds(new ArrayList<String>(KmDefaultSettingPreference.getInstance().getDefaultAgentIds()));
+            }
+            if(KmDefaultSettingPreference.getInstance().getDefaultAssignee() != null) {
```

Comment:
create a single instance for `KMDefaultSettingPreference.getInstance()` & use it for all.